src/chains/tests/test_models.py

```diff
@@ -31,7 +56,9 @@ def test_oracle_gas_parameter_with_null_url(self):
 
     def test_null_oracle_gas_parameter_with_url(self):
         chain = ChainFactory.create(
-            gas_price_oracle_url=self.faker.url(), gas_price_oracle_parameter=None
+            gas_price_oracle_url=self.faker.url(),
+            gas_price_oracle_parameter=None,
+            gas_price_fixed_wei=None,
```

Comment:
These are outwardly exposed?

Reply:
Yes. Depending on the type of information available it'll either expose the oracle data or the fixed price data. (this is more a representation of the DB table – we avoided creating a new table with foreign keys just for gas related data)